src/carcockpit/gui.hpp

```diff
@@ -22,6 +22,7 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #pragma once
 
 #include <ruis/widgets/proxy/key_proxy.hpp>
+#include <ruis/widgets/proxy/mouse_proxy.hpp>
```

Comment:
mouse proxy is not used in this file, remove the include

Reply:
done